programs/marinade-finance/src/instructions/crank/stake_reserve.rs

```diff
@@ -317,4 +317,24 @@ impl<'info> StakeReserve<'info> {
         });
         Ok(())
     }
+
+    pub fn return_unused_stake_account_rent(
```

Comment:
We're doing this in several places, how do you feel about creating an utility function `fn resturn_rent_unused_stake_account(stake_account, stake_history, stake_program) -> Result()` and use it everywhere instead of creating the same in several places? (a trait or a common struct sounds like too much)

Reply:
It is using clock, stake_history and stake_program account infos so too much parameters. Better to have self as Accounts struct but it is a big pain to reuse parts of the accounts struct